### PR TITLE
install: pass the 'shaman' to Shaman class

### PR DESCRIPTION
**Pull request by lxbsz**
Signed-off-by: Xiubo Li <xiubli@redhat.com>

### CODE REVIEW

**Comment by lxbsz** [comment]
Fixing the error in https://github.com/ceph/ceph/pull/37138#issuecomment-741827256.

It didn't pass the
```
shaman:
   force_shaman: True
```
down to the Shanman Class.

File: teuthology/task/install/__init__.py [hunk 1]
```diff
@@ -607,6 +607,7 @@ def task(ctx, config):
                 sha1=config.get('sha1'),
                 tag=config.get('tag'),
                 wait_for_package=config.get('wait_for_package', False),
+                shaman=config.get('shaman', {})
```
Comment (dillaman) [inline teuthology/task/install/__init__.py:610]: Nit: perhaps treat this more like the `repos` key below:

```
if 'shaman' in config:
    nested_config['shaman'] = config['shaman']
```

Comment (lxbsz) [reply teuthology/task/install/__init__.py]: Done.

**Review by dillaman — APPROVED** [review]
lgtm

**Comment by kshtsk** [comment]
interesting....

**Comment by kshtsk** [comment]
@susebot run deploy

**Pull request merged**